### PR TITLE
Adds ambassador integration for Central UI

### PR DESCRIPTION
**Pull request by swiftdiaries**
Addresses: #528

1) Adds ambassador integration for centraldashboard
2) Routes centraldashboard at "/" so that it becomes the homepage.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/779)

### CODE REVIEW

**Comment by jlewi** [comment]
Can you fix the lint issues?

File: kubeflow/core/centraldashboard.libsonnet [hunk 1]
```diff
@@ -2,6 +2,9 @@
   // TODO(https://github.com/ksonnet/ksonnet/issues/222): Taking namespace as an argument is a work around for the fact that ksonnet
   // doesn't support automatically piping in the namespace from the environment to prototypes.
 
+  // TODO(https://github.com/kubeflow/kubeflow/issues/527): The central UI is currently using a personal docker image: swiftdiaries/centraldashboard:0.3
```
Comment (jlewi) [inline kubeflow/core/centraldashboard.libsonnet]: In a separate PR please create a Makefile like:
https://github.com/kubeflow/kubeflow/blob/master/bootstrap/Makefile

To build and push the image. I will then run it in order to push the image to gcr.io/kubeflow-images-public.

Comment (swiftdiaries) [reply kubeflow/core/centraldashboard.libsonnet]: Yeah, sure thing. Will get it done.

**Comment by jlewi** [comment]
Looks like minikube test failed.

/test all

**Comment by jlewi** [comment]
@swiftdiaries ping? Would be great to get this submitted today.

**Pull request merged**